task_processing/interfaces/event.py

```diff
@@ -38,7 +31,7 @@ def __invariant__(r): return (
     # use time.time() to generate
     timestamp = field(type=float)
     # reference to platform-specific event object
-    raw = field(initial=addict.Dict())
+    raw = field(mandatory=True)
```

Comment:
I think the raw field is supposed to be of type addict.Dict. Can you add the type here?

Reply:
sorry, didn't notice this comment before replying to the other one: https://github.com/Yelp/task_processing/pull/112/files/e83d65482c875803a43536525d4b1d393ad0e028#r200470432